src/hyperfine/internal.rs

```diff
@@ -97,6 +103,39 @@ pub fn min(vals: &[f64]) -> f64 {
         .unwrap()
 }
 
+pub fn write_benchmark_comparison(results: &Vec<ExportEntry>) {
+    if results.len() < 2 {
+        return;
+    }
+
+    // Show which was faster, maybe expand to table later?
+    let mut fastest_item: &ExportEntry = &results[0];
+    let mut longer_items: Vec<&ExportEntry> = Vec::new();
+
+    for run in &results[1..] {
+        if let Some(Ordering::Less) = fastest_item.mean.partial_cmp(&run.mean) {
+            longer_items.push(run);
+        } else {
+            longer_items.push(fastest_item);
+            fastest_item = run;
+        }
+    }
+
+    println!("{}\n", "Result Comparison".bold());
+    println!("'{}' ran", fastest_item.command.green());
```

Comment:
Maybe indent this by two spaces (similar to the `Benchmark #..` blocks)?

Reply:
Also, maybe refer to the number of the benchmark here (if there are only subtle differences between the benchmark commands, it might be easier to identify them by number, because that's the order they were entered in). Maybe like this:
```
'sleep 0.2' (#1) ran
    1.49x faster than 'sleep 0.3' (#2)
    1.98x faster than 'sleep 0.4' (#3)
```

Reply:
I can do this, I'll have to map them to an intermediate type to retain the indices, but it should be fine

Reply:
If you are not yet working on this.. maybe leave this one out. I'm currently working on wrapping commands in a separate type anyway.